lib/puppet/provider/group/directoryservice.rb

```diff
@@ -9,4 +9,10 @@
   confine :operatingsystem => :darwin
   defaultfor :operatingsystem => :darwin
   has_feature :manages_members
+
+  def members_insync?(current, should)
+  	return false unless current
```

Comment:
Nitpick, but it looks like the surrounding tab width is 2 and not 4.  We generally don't care about tab width so long as it is consistently applied in a file (assuming it uses spaces and not tabs).